boot.php

```diff
@@ -134,28 +134,6 @@
  */
 define('MAX_LIKERS',    75);
 
-/**
- * @name Notify
- *
- * Email notification options
- * @{
- */
-define('NOTIFY_INTRO',        1);
```

Comment:
Shouldn't we reduce step by step the `boot.php` until we can delete it completely? Making old things deprecated doubles the code instead of reduces it :) That's why I searched step by step each constant everywhere in the codebase

Reply:
The idea is to leave the constants for one stable version before removing them, just to be sure. I haven't been keeping up with this and as a result `boot.php` contains a bunch of unused deprecated constants but they can safely be removed at any point now because we went through that safety period.

Reply:
OK, so what about `@deprecated 2020.03` so we can safely loop over the codebase each new release if we found things 2 releases later and remove these constants? 2 releases should be enough to prove we found all occurrences